src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -22,8 +22,7 @@ def write_caption_data_to_file(caption_data: dict):
     # mm.dd.YY_H:M:S
     dt_string = now.strftime("%m.%d.%Y_%H:%M:%S")
 
-    # path = pathlib.Path(__file__).resolve().parent.parent.parent.parent
-    path = config.path
+    path = config.path_to_perception
     path = str(path) + '/data/byte_pairs/'
```

Comment:
There's not really a reason to define path, and then use path again. Could you take out `path = config.path_to_perception` and instead change line 26 to be `path = str(config.path_to_perception) + '/data/byte_pairs/'`

Reply:
i'm really curious why the creator of this library doesn't return a file path as a string